### ISSUE TITLE
Autopep Modifies Python 3.8 Assignment Operator Incorrectly

### ISSUE THREAD

**Issue by chrisgilbert** [hhatto/autopep8#542]
I started using the new `:=` assignment operations/walrus operator [from python 3.8 ](https://www.python.org/dev/peps/pep-0572/)today. When I ran the file through autopep, it unfortunately treated the `:` as the end of the statement and added a newline, breaking the code syntax. Here's a test case to reproduce the problem.

---

## Python Code
```python
sql_stmt = ""
with open(filename) as f:
    while line := f.readline():
        sql_stmt += line
```

## Command Line and Configuration
Default config.

Command Line
```shell
$ autopep8 
sql_stmt = ""
with open(filename) as f:
     while line:
         = f.readline():
         sql_stmt += line
```

## Your Environment
* Python version: 3.8.2
* autopep8 version: 1.5.2
* Platform: MacOSX 10.14.4 (Mojave)

**Comment by hhatto** [comment]
Duplicated #507

The pycodestyle 2.6.0a1 (pre-release) has been released, so you can use it to work around it.
Execute `pip install --upgrade pycodestyle==2.6.0a1` and then execute autopep8.

```
$ pip install autopep8
$ pip install pycodestyle==2.6.0a1
$ autopep8 --version
autopep8 1.5.3 (pycodestyle: 2.6.0a1)
$ cat t.py
sql_stmt = ""
with open(filename) as f:
    while line := f.readline():
        sql_stmt += line
$ autopep8 t.py
$
```

**Comment by hhatto** [comment]
autopep8 1.5.3 has been released to support pycodestyle 2.6.0.
Please use autopep8 1.5.3 or higher version.
